File: id and ship.cpp
#include <iostream>
#include <cstring>
using namespace std;

int main() {
	int testCase;
	cin >> testCase;
	for(int i=0;i<testCase;i++)
        {
            char userId;
            cin >> userId ;
            char finalId = tolower(userId);
            switch(finalId)
            {
                case 'b':
                    cout<<"BattleShip\n ";
                    break;
                case 'c':
                    cout<<"Cruiser\n ";
                    break;
                case 'd':
                    cout<<"Destroyer\n ";
                    break;
                case 'f':
                    cout<<"Frigate\n ";
                    break;
            }
        }
    return 0;
}

File: id and ship.java
/* package codechef; // don't place package name! */

import java.util.Scanner;
import java.lang.*;
import java.io.*;

/* Name of the class has to be "Main" only if the class is public. */
class Codechef
{
	public static void main (String[] args) throws java.lang.Exception
	{
        String finalId;
        Scanner myObj = new Scanner(System.in);
        int testCase = myObj.nextInt();
        for(int i=0;i<=testCase;i++)
        {
            String userId = myObj.nextLine();
            finalId = userId.toLowerCase();
            switch(finalId)
            {
                case "b":
                    System.out.println("BattleShip");
                    break;
                case "c":
                    System.out.println("Cruiser");
                    break;
                case "d":
                    System.out.println("Destroyer");
                    break;
                case "f":
                    System.out.println("Frigate");
                    break;
            }
        }
    }
}
